packages/autocomplete-plugin-recent-searches/src/createRecentSearchesPlugin.ts

```diff
@@ -50,19 +52,36 @@ export function createRecentSearchesPlugin({
             return store.getAll();
           },
           templates: {
-            item({ item }) {
-              return `
-                <div class="autocomplete-item">
-                  <div>
-                    <svg viewBox="0 0 22 22" width="16" height="16" fill="currentColor">
-                      <path d="M0 0h24v24H0z" fill="none"/>
-                      <path d="M13 3c-4.97 0-9 4.03-9 9H1l3.89 3.89.07.14L9 12H6c0-3.87 3.13-7 7-7s7 3.13 7 7-3.13 7-7 7c-1.93 0-3.68-.79-4.94-2.06l-1.42 1.42C8.27 19.99 10.51 21 13 21c4.97 0 9-4.03 9-9s-4.03-9-9-9zm-1 5v5l4.28 2.54.72-1.21-3.5-2.08V8H12z"/>
-                    </svg>
+            item({ item, root }) {
+              if (!root.classList.contains('aa-RecentSearchesItem')) {
```

Comment:
If you were using a framework like Vue or React, you wouldn't do this but rather create a sub-container in the `root`. Why mutate the class of the root here then?

Reply:
You're right. `aa-Item` was a flex container, so I tried to leverage it. But actually I was uncomfortable doing that.
https://github.com/algolia/autocomplete.js/pull/326/commits/2da7b94eda8c55ea99e05529c089e15f28ff00cb